components/dropzone/stories/dropzone.stories.js

```diff
@@ -49,16 +50,25 @@ export default {
 };
 
 export const Default = DropzoneGroup.bind({});
-Default.args = {
-	description: [
-		() => {
-			return html`${Link({ url: "#", text: "Select a file" })} from your computer.`;
-		},
-	],
-	label: "Drop file to replace",
-	customStyles: {
-		width: "300px",
-	},
+Default.args = {};
+
+export const Dragged = Template.bind({});
+Dragged.args = {
+	isDragged: true,
+};
+Dragged.tags = ["!dev"];
+Dragged.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const Filled = Template.bind({});
```

Comment:
Is this one only medium or does it come in different sizes? If it's only meant to be one size, would you want to add that in here so it's documented?